README.md

```diff
@@ -21,6 +22,13 @@ Fusion Bites is a unique and innovative restaurant that celebrates the fusion of
 
 [Back to Top ⬆️](#top)
 
+<h2 id="#description">🥘 Project Description </h2>
+Watch this video for more details:
+
+https://www.loom.com/share/7692e2ccdf5242d2bdfbbdbc7a521088?sid=c2a79393-4863-470f-873d-ea930afe5473
+
```

Comment:
- [ ] You did a great job recording a video, unfortunately, the video does not meet the requirements, such as every collaborator appearing in the video, highlighting a piece of code you both are proud of, etc.


![video](https://github.com/Kaiserabbas/Fusion-Bites/assets/53124548/241363c5-d1d8-4759-a9d8-57fb9608985a)

Reply:
We have recorded a video with both parts present and explaining the project.